release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+the scope of an existing ingress controller will temporarily impact ingress
+availability.
+
+[id="ocp-4-2-kubernetes-cni-plug-in-additions"]
+Kubernetes CNI plug-in additions and enhancements
+
+Several Kubernetes CNI plug-ins are added or enhanced in {product-title} 4.2 to
+grow capability.
+
+These SR-IOV solutions remain in Technology Preview in {product-title} 4.2:
+
+* RDMA and RoCE Support
+* DPDK Mode for SR-IOV VFs
+* Admission Controller
+* Operator
+
+New CNI plug-ins:
+
+* IPVLAN
+* Bridge with VLAN
+* Static IPAM
+
+[id="ocp-4-2-enablement-of-GPUs"]
+==== Enablement of GPUs in an OpenShift cluster
+
+GPUs are now supported on Red Hat Enterprise Linux (RHEL) 7 nodes. They are not
+supported on RHEL CoreOS nodes because of lack of support for CUDA driver and
+driver containers.
+
+[id="ocp-4-2-web-console"]
+=== Web console
+
+[id="ocp-4-2-custom-branding"]
+==== Console customization options
+
+You can customize the {product-title} web console to set a custom logo, links,
+notification banners, and command line downloads. This is especially helpful if you
+need to tailor the web console to meet specific corporate or government
+requirements.
+
+See Customizing the web console for more information.
+
+[id="ocp-4-2-new-api-explorer"]
+==== New API Explorer
+
+You can now easily search and manage API resources in the *Explore API
+Resources* dashboard located at *Home* -> *Explore*.
+
+View the schema for each API and what parameters being supported, manage the
+instances of the API, and review the access of each API.
+
+[id="ocp-4-2-machine-autoscaler"]
+==== Machine Autoscaler
+
+You can now scale your cluster with the Machine Autoscaler. The Machine
+Autoscaler adjusts the number of Machines in the MachineSets being deployed in
+your cluster. Increase Machines when the cluster runs out of resources to
+support more deployments.  Any changes, such as the minimum or maximum number of
+instances, are immediately applied to the MachineSet that MachineAutoscalers
+target.
+
+[id="ocp-4-2-developer-perspective"]
+==== Developer Perspective
+
+The Developer perspective adds a developer-focused perspective to the web
+console. It provides workflows specific to developer use cases, such as creation
+and deployment of applications to {product-title} using multiple options. It
+provides a visual representation of the applications within a project, their
+build status, and the components and services associated with them,  enabling
+easy interaction and monitoring. It incorporates Serverless capabilities
+(Technology Preview) and the ability to create workspaces to edit your
+application code using Eclipse Che.
+
+[id="ocp-4-2-prometheus-queries"]
+==== Prometheus queries
+
+You can now run Prometheus queries directly in the web console.
+
+[id="ocp-4-2-general-web-console-updates"]
+==== General web console updates
+
+* The dashboard is redesigned with more metrics.
+* *Catalog* is moved.
```

Comment:
is moved to the dev perspective, right @spadgett? should probably say where it was moved to.

Reply:
Yes, it's moved to the Developer perspective.

Developer -> Add+ -> From Catalog